index.html

```diff
@@ -84,197 +46,249 @@ <h4>Zero-ops OpenStack on just about any Linux box.</h4>
+        <img src="https://assets.ubuntu.com/v1/53c3be42-linuc-logo.svg" style="height: 90px; margin-top: 1rem;" alt="">
+        <h4 class="p-heading--five p-tab__title">Linux</h4>
+      </a>
 
-          <p>
-            Install Multipass then start a multipass vm:
-          </p>
-          <div class="p-code-copyable">
-            <input class="p-code-copyable__input" value="multipass launch --name microkstack-vm --mem 8G --disk 40G" readonly="readonly">
-            <button class="p-code-copyable__action">Copy to clipboard</button>
-          </div>
+      <a id="tab-two" href="#tab-two__content"
+        class="col-small-2 col-medium-2 col-4 p-card p-tab u-align--center u-vertically-center js-tab" role="tab">
+        <img src="https://assets.ubuntu.com/v1/11ecbd2d-2018-logo-windows.svg" style="height: 145px;" alt="Windows">
+      </a>
 
-          <p>
-            Install the MicroStack snap on that multipass vm:
-          </p>
-          <div class="p-code-copyable u-sv3">
-            <input class="p-code-copyable__input" value="multipass exec microkstack-vm -- sudo snap install microstack --classic --edge" readonly="readonly">
-            <button class="p-code-copyable__action">Copy to clipboard</button>
-          </div>
+      <a id="tab-three" href="#tab-three__content"
+        class="col-small-2 col-medium-2 col-4 p-card p-tab u-align--center u-vertically-center js-tab" role="tab">
+        <img src="https://assets.ubuntu.com/v1/236f314d-macos.svg" style="height: 145px;" alt="macOS">
+      </a>
+    </div>
 
-          <h3>Configure networks and OpenStack databases</h3>
+    <div class="row">
+      <div class="col-6 js-tab__content" id="tab-one__content" role="tabpanel" aria-labelledby="tab-one">
+        <ol class="p-stepped-list u-no-margin--left">
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Install the microkstack snap</h3>
 
-          <div class="p-code-copyable">
-            <input class="p-code-copyable__input" aria-label="sudo microstack.init" value="sudo microstack.init --auto" readonly="readonly">
-            <button class="p-code-copyable__action">Copy to clipboard</button>
-          </div>
+            <div class="p-code-copyable u-stepped-list-margin">
+              <input class="p-code-copyable__input" aria-label="sudo snap install microk8s --classic"
+                value="sudo snap install microstack --classic --edge" readonly="readonly">
+              <button class="p-code-copyable__action">Copy to clipboard</button>
+            </div>
 
-          <p>
-            That’s it! Your OpenStack is ready.
-          </p>
-        </div>
+            <p class="p-stepped-list__content u-stepped-list-margin">Don’t have the <code>snap</code> command? <a
+                href="https://docs.snapcraft.io/installing-snapd" class="p-link--external">Get set up for snaps.</a></p>
+          </li>
```

Comment:
As we discussed yesterday, this should continue with the two following steps from the current site:
(2) Configure networks and OpenStack databases
(3) Launch a VM